CODEOWNERS

```diff
@@ -1 +1 @@
-* @marinalimeira @infraredgirl @robmorgan @ina-stoyanova @gruntwork-io/maintenance-tier-3-orion
+* @marinalimeira  @robmorgan @ina-stoyanova @gruntwork-io/maintenance-tier-3-orion
```

Comment:
There is an extra space here but from this PR's reviewers, I don't think this affects anything.